sympy/parsing/sympy_parser.py

```diff
@@ -989,7 +989,15 @@ def parse_expr(s, local_dict=None, transformations=standard_transformations,
         raise TypeError('expecting local_dict to be a dict')
 
     if global_dict is None:
-        global_dict = {}
+        from sympy.core.power import Pow
+        from sympy.functions.elementary.complexes import Abs
+        from sympy.functions.elementary.miscellaneous import Max, Min
+        global_dict = {  # map built-in functions to sympy functions
+            'abs': Abs,
+            'max': Max,
+            'min': Min,
+            'pow': Pow,
```

Comment:
This should be `pow`, which will still work because sympy types define `__pow__`, but will also support the 3-argument version.